packages/expo-camera/android/src/main/java/expo/modules/camera/next/ExpoCameraView.kt

```diff
@@ -280,6 +281,7 @@ class ExpoCameraView(
 
   private fun createImageAnalyzer(): ImageAnalysis =
     ImageAnalysis.Builder()
+      .setTargetResolution(Size(1920, 1080))
```

Comment:
```suggestion
  ImageAnalysis.Builder()
      .setResolutionSelector(
        ResolutionSelector.Builder()
          .setResolutionStrategy(ResolutionStrategy.HIGHEST_AVAILABLE_STRATEGY)
          .build()
      )
```
We shouldn't hardcode the resolution and this method is deprecated. We should do this